pkg/backend/display/progress.go

```diff
@@ -1059,15 +1059,18 @@ func (display *ProgressDisplay) getStepDoneDescription(step engine.StepEventMeta
 	return op.Color() + getDescription() + colors.Reset
 }
 
-func (display *ProgressDisplay) getPreviewText(op deploy.StepOp) string {
-	switch op {
+func (display *ProgressDisplay) getPreviewText(step engine.StepEventMetadata) string {
+	switch step.Op {
 	case deploy.OpSame:
 		return "no change"
 	case deploy.OpCreate:
 		return "create"
 	case deploy.OpUpdate:
 		return "update"
 	case deploy.OpDelete:
+		if step.Old.Delete {
+			return "delete from previous plan"
```

Comment:
We should not use the term "plan" here, as it is not a concept that we otherwise expose to the user. How about "delete from previous replacement"?

Reply:
Sounds fine to me. What I was going for was something that indicated that this operation is happening because it didn't happen the previous time an update happened.

Reply:
Yeah, we definitely want to communicate that. @joeduffy any ideas here?

Reply:
Ping (cc @lukehoban) - any last thoughts on the phrasing here? I'm going to go with Pat's suggestion `delete from previous replacement` if there aren't any other objections.